nixos/modules/services/search/manticore.nix

```diff
@@ -0,0 +1,78 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.manticore;
+
+in {
+
+  options = {
+    services.manticore = {
+
+      enable = mkEnableOption (mdDoc "Manticore Search full-text search solution");
+
+      configFile = mkOption {
+        type = types.path;
+        default = "${pkgs.manticoresearch}/etc/manticoresearch/manticore.conf";
+        defaultText = literalExpression ''"''${pkgs.manticoresearch}/etc/manticoresearch/manticore.conf"'';
+        description = lib.mdDoc "Path to Manticore configuration file.";
+      };
+
+    };
+  };
+
+  config = mkIf cfg.enable {
+
+    systemd = {
+      services.manticore = {
+        wantedBy = [ "multi-user.target" ];
+        description = "Manticore Search full-text search solution";
+        after = [ "network.target" ];
+        serviceConfig = {
+          # Took config from upstream service https://github.com/manticoresoftware/manticoresearch/blob/master/dist/deb/manticore.service.in
```

Comment:
Please use the upstream systemd file and add our changes as a dropin which should be done automatically.